docs/guide/events/reading/aggregating.md

```diff
@@ -124,8 +124,276 @@ var invoice = await theSession.Events.AggregateStreamAsync<Invoice>(invoiceId);
+);
+
+public record RoomBooked(
+    Guid HotelId,
+    RoomType RoomType
+);
+
+public record GuestCheckedOut(
+    Guid HotelId,
+    Guid GuestId,
+    RoomType RoomType
+);
+
+public class RoomsAvailability
+{
+    public Guid Id { get; private set; }
+
+    public int AvailableSingleRooms => roomTypeCounts[RoomType.Single];
+    public int AvailableDoubleRooms => roomTypeCounts[RoomType.Double];
+    public int AvailableKingRooms => roomTypeCounts[RoomType.King];
+
+    private Dictionary<RoomType, int> roomTypeCounts = new ();
+
+    public void Apply(HotelRoomsDefined @event)
+    {
+        Id = @event.HotelId;
+        roomTypeCounts = @event.RoomTypeCounts;
+    }
+
+    public void Apply(RoomBooked @event)
+    {
+        roomTypeCounts[@event.RoomType] -= 1;
+    }
+
+    public void Apply(GuestCheckedOut @event)
+    {
+        roomTypeCounts[@event.RoomType] += 1;
+    }
+}
+```
+<sup><a href='https://github.com/JasperFx/marten/blob/master/src/Marten.Testing/Events/Aggregation/time_travelling_samples.cs#L12-L64' title='Snippet source file'>snippet source</a> | <a href='#snippet-sample_aggregate-stream-time-travelling-definition' title='Start of snippet'>anchor</a></sup>
+<!-- endSnippet -->
+
+**You can get the stream state at the point of time, providing a timestamp:**
+
+<!-- snippet: sample_aggregate-stream-time-travelling-by-point-of-time -->
+<a id='snippet-sample_aggregate-stream-time-travelling-by-point-of-time'></a>
+```cs
+var roomsAvailabilityAtPointOfTime =
+    await theSession.Events
+        .AggregateStreamAsync<RoomsAvailability>(hotelId, timestamp: pointOfTime);
+```
+<sup><a href='https://github.com/JasperFx/marten/blob/master/src/Marten.Testing/Events/Aggregation/time_travelling_samples.cs#L123-L129' title='Snippet source file'>snippet source</a> | <a href='#snippet-sample_aggregate-stream-time-travelling-by-point-of-time' title='Start of snippet'>anchor</a></sup>
+<!-- endSnippet -->
+
+
+**Or specific version:**
+
+<!-- snippet: sample_aggregate-stream-time-travelling-by-specific-version -->
+<a id='snippet-sample_aggregate-stream-time-travelling-by-specific-version'></a>
+```cs
+var roomsAvailabilityAtVersion =
+    await theSession.Events
+        .AggregateStreamAsync<RoomsAvailability>(hotelId, version: specificVersion);
+```
+<sup><a href='https://github.com/JasperFx/marten/blob/master/src/Marten.Testing/Events/Aggregation/time_travelling_samples.cs#L137-L143' title='Snippet source file'>snippet source</a> | <a href='#snippet-sample_aggregate-stream-time-travelling-by-specific-version' title='Start of snippet'>anchor</a></sup>
+<!-- endSnippet -->
+
+
+## Aggregating stream into state
+
+Marten also allows aggregating the stream into a specific entity instance. That means that it will take a particular set of events and apply it one by one to an object. To achieve that, you should pass the base entity state as a `state` parameter into the `AggregateStream` method.
```

Comment:
Change `That means that it will take a...` -> `This means that a particular set of events are taken and applied to an object one by one in the same order of occurrence. To achieve this, you should...`

Reply:
Fixed in https://github.com/JasperFx/marten/pull/1907/commits/60c889e260e671da1c421443337ff04e6e2acb48 👍